calculator_test.go

```diff
@@ -7,18 +7,59 @@ import (
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
+	type testCase struct {
```

Comment:
Good! In fact, we often leave this struct anonymous, since it's only temporary. We might write, for example:

```go
testCases := []struct{
		a, b float64
		want float64
	}{
		{a: 2, b: 2, want: 4},
		...
	}
}
```